baybe/surrogates/base.py

```diff
@@ -65,42 +80,103 @@ class Surrogate(ABC, SerialMixin):
     """Callable preparing surrogate inputs for training/prediction.
 
     Transforms a dataframe containing parameter configurations in experimental
-    representation to a corresponding dataframe containing their computational
-    representation. Only available after the surrogate has been fitted."""
+    representation to a corresponding dataframe containing their **scaled**
+    computational representation. Only available after the surrogate has been fitted."""
 
-    _target_transform: Callable[[pd.DataFrame], pd.DataFrame] | None = field(
+    _output_transform: Callable[[pd.DataFrame], pd.DataFrame] | None = field(
         init=False, default=None, eq=False
     )
-    """Callable preparing surrogate targets for training.
+    """Callable preparing surrogate outputs for training.
 
     Transforms a dataframe containing target measurements in experimental
-    representation to a corresponding dataframe containing their computational
-    representation. Only available after the surrogate has been fitted."""
+    representation to a corresponding dataframe containing their **scaled**
+    computational representation. Only available after the surrogate has been fitted."""
+
+    # TODO: type should be `Standardize | _NoTransform`` but is currently
+    #   omitted due to: https://github.com/python-attrs/cattrs/issues/531
+    _output_scaler = field(init=False, default=None, eq=False)
+    """Optional callable for scaling output values.
+
+    Scales a tensor containing target measurements in computational representation
+    to make them ready for processing by the surrogate. Only available after the
+    surrogate has been fitted."""
 
     def to_botorch(self) -> Model:
         """Create the botorch-ready representation of the model."""
         from baybe.surrogates._adapter import AdapterModel
 
         return AdapterModel(self)
 
-    def transform_inputs(self, data: pd.DataFrame) -> pd.DataFrame:
+    @staticmethod
+    def _make_parameter_scaler(
+        parameter: Parameter,
+    ) -> ParameterScalerProtocol | Literal["passthrough"]:
```

Comment:
Is the literal really needed here? Its a bit more userfriendly to allow just returning `None` if no scaling should be done and this being correctly converted to `passthrough` in `_make_input_scaler`
It would also be consistent with `_make_output_scaler`

Reply:
good idea, done